src/GitVersionExe/Program.cs

```diff
@@ -129,17 +129,27 @@ static void ConfigureLogging(Arguments arguments)
             {
                 try
                 {
-                    Directory.CreateDirectory(Path.GetDirectoryName(arguments.LogFilePath));
-                    if (File.Exists(arguments.LogFilePath))
+                    var logFileFullPath = Path.GetFullPath(arguments.LogFilePath);
+                    var logFile = new FileInfo(logFileFullPath);
+
+                    if (logFile.Directory == null)
+                        throw new DirectoryNotFoundException(String.Format("The directory of {0} does not exist.", logFile));
+
+                    if (!logFile.Directory.Exists)
```

Comment:
`logFile.Directory.Create();` doesn't throw if the directory doesn't exist. Can just remove the if